metadata.json

```diff
@@ -36,6 +50,10 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 2.4.0 <= 4.5.0"
+    },
+    {
+      "name": "stahnma/epel",
+      "version_requirement": ">= 1.0.0"
```

Comment:
Could you add "<= 1.0.2" to the end of this line?  Metadata guidelines suggest a maximum version must be specified.